tests/sql_parse_tests.py

```diff
@@ -310,3 +310,23 @@ def test_explain(self):
         self.assertEquals(True, sql.is_explain())
         self.assertEquals(False, sql.is_select())
         self.assertEquals(True, sql.is_readonly())
+
+    def test_complex_extract_tables(self):
```

Comment:
can we add another test that does not use `ON`? Something like
```
SELECT * 
FROM table_a AS a, table_b AS b
WHERE a.id = b.id
```

Reply:
Done